pkgs/by-name/ti/tint2/package.nix

```diff
@@ -78,6 +78,9 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
+    substituteInPlace src/tint2conf/CMakeLists.txt \
+      --replace-fail "RSVG_LIBRARIES} )" "RSVG_LIBRARIES} m)"
+
     for f in ./src/launcher/apps-common.c \
              ./src/launcher/icon-theme-common.c
     do
```

Comment:
substituteInPlace can edit multiple files at the same time

Reply:
Thanks, it was like this forever, but I cleaned it up.